og_ui/tests/src/Functional/BundleFormAlterTest.php

```diff
@@ -139,4 +147,21 @@ protected function assertTargetBundles($expected, $message) {
     $this->assertEquals($expected, $settings['target_bundles'], $message);
   }
 
+  /**
+   * Checks whether the target entity type in the group content is as expected.
+   *
+   * @param string $expected
+   *   The expected target entity type.
+   * @param string $message
+   *   The message to display with the assertion.
+   */
+  protected function assertTargetType($expected, $message) {
+    /** @var EntityFieldManagerInterface $entity_field_manager */
+    $entity_field_manager = $this->container->get('entity_field.manager');
+    $entity_field_manager->clearCachedFieldDefinitions();
+    $field_definitions = $entity_field_manager->getFieldStorageDefinitions('node');
+    $setting = $field_definitions['og_group_ref']->getSetting('target_type');
```

Comment:
I usually prefer to avoid hardcoding `'og_group_ref'` - but it won't hold this PR